api/openapi-spec/v1.0.yaml

```
@@ -493,6 +493,40 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+  '/groups/{group-id}/members':
+    get:
+      tags:
+        - group
+      summary: Get a list of a groups members
```

copy the description of https://learn.microsoft.com/en-us/graph/api/group-list-members?view=graph-rest-1.0&tabs=http
```suggestion
      summary: Get a list of the group's direct members.
```

the text goes on:
> Get a list of the group's direct members. A group can have users, organizational contacts, devices, service principals and other groups as members. Currently service principals are not listed as group members due to staged roll-out of service principals on Graph V1.0 endpoint. This operation is not transitive.

>
> When a group contains more than 100 members, Microsoft Graph returns a @odata.nextLink property in the response that contains a URL to the next page of results. If that property is present, continue making additional requests with the @odata.nextLink URL in each response, until all the results are returned, as described in [paging Microsoft Graph data in your app](https://learn.microsoft.com/en-us/graph/paging).

but that might be overkill ... even the pagination note is something for later